src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -1,16 +1,40 @@
 package com.minecolonies.colony;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
 import com.minecolonies.MineColonies;
+import com.minecolonies.achievements.ModAchievements;
 import com.minecolonies.colony.buildings.AbstractBuilding;
 import com.minecolonies.colony.buildings.BuildingHome;
 import com.minecolonies.colony.buildings.BuildingTownHall;
 import com.minecolonies.colony.materials.MaterialSystem;
 import com.minecolonies.colony.permissions.Permissions;
 import com.minecolonies.configuration.Configurations;
 import com.minecolonies.entity.EntityCitizen;
-import com.minecolonies.network.messages.*;
+import com.minecolonies.network.messages.ColonyViewBuildingViewMessage;
+import com.minecolonies.network.messages.ColonyViewCitizenViewMessage;
+import com.minecolonies.network.messages.ColonyViewMessage;
+import com.minecolonies.network.messages.ColonyViewRemoveBuildingMessage;
+import com.minecolonies.network.messages.ColonyViewRemoveCitizenMessage;
+import com.minecolonies.network.messages.PermissionsMessage;
 import com.minecolonies.tileentities.TileEntityColonyBuilding;
-import com.minecolonies.util.*;
+import com.minecolonies.util.BlockPosUtil;
+import com.minecolonies.util.EntityUtils;
+import com.minecolonies.util.LanguageHandler;
+import com.minecolonies.util.Log;
+import com.minecolonies.util.MathUtils;
+import com.minecolonies.util.ServerUtils;
+import com.minecolonies.util.Utils;
+
+import com.sun.corba.se.spi.activation.Server;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1191)